content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc

```diff
@@ -0,0 +1,60 @@
+
+         * yubihsm-cngprovider-windows-amd64.msi (YubiHSM Key Storage Provider)
+         * yubihsm-connector-windows-amd64.msi (YubiHSM Connector for Windows)
+
+
+=== Default YubiHSM 2 Default Device Configuration
+
+The YubiHSM 2 device comes with a default, factory-installed authentication key that has the password `password`. As part of the configuration in this guide, the default authentication key will be destroyed. If the device is reset to its default configuration, any objects stored on the device that are not factory-installed are destroyed. Reset instructions can be found in https://developers.yubico.com/YubiHSM2/Usage_Guides/Factory_reset.html[Factory Reset].
+
+To ensure that the device has not been tampered with, check that the YubiHSM 2 device has the default configuration.  To verify that the YubiHSM 2 is in default configuration, follow the steps below:
+
+*Step 1* If the application that calls the YubiHSM Connector is *running on a local host*, the Connector can be started with the command `yubihsm-connector` without additional parameters. In Windows Server 2012 SP2 or higher, `yubihsm-connector.exe` is located in `C:\Program Files\YubiHSM Connector\`.
+
+         If the application is *running on a VM or a different server*, start the YubiHSM Connector on the host operating system in networking mode. For example, if the host machine’s IP address is 192.168.100.252, launch the Connector on the host OS with the following command:
+
+         `$ yubihsm-connector -l 192.168.100.252:12345`
+
+	       [TIP]
+         =====
+         For testing or debugging the YubiHSM Connector, the flag `-d` can be set.
+         =====
+
+*Step 2* To gain shell access to the YubiHSM 2, launch the YubiHSM Shell program by opening a Command Prompt and running the command `yubihsm-shell`. If a networked Connector is used, set the parameter `--connect <connector URL>`, for example:
+
+         $ yubihsm-shell --connector 192.168.100.252:12345
+
+         [TIP]
+         =====
+         For testing or debugging the YubiHSM Shell, the flag `-d` can be set.
+         =====
+
+*Step 3* To connect to the YubiHSM 2, at the `yubihsm` command line, type `connect`. A message saying that you have a successful connection is displayed.
+
+*Step 4* To open a session with the YubiHSM 2, type `session open 1` (where 1 is the ID of the default authentication key pre-installed on the device).
+
+*Step 5* Type in the default password: `password`. A message confirming  that the session has been set up successfully is displayed.
+
+*Step 6* You now have an administrative connection to the YubiHSM 2 and can list the objects available. To list the objects, type `list objects 0` and press Enter. Your results should be similar to the following:
+
+         `id: 0x0001, type: authentication-key, sequence: 0`
+
+         To exit, type `quit`.
+
+
+
+link:Configure_the_Primary YubiHSM_2_Device.html[Next: Configure the Primary YubiHSM 2 Device]
```

Comment:
Please change this to be a link to the adoc file, then it works for both html and adoc.

Reply:
Done throughout